src/columnstore/execution/columnstore_insert.hpp

```diff
@@ -0,0 +1,64 @@
+// Implementation of insertion physical operation with column storage.
+
+#pragma once
+
+#include "columnstore/columnstore.hpp"
+#include "columnstore/columnstore_table.hpp"
+#include "duckdb/common/enums/operator_result_type.hpp"
+#include "duckdb/common/unique_ptr.hpp"
+#include "duckdb/planner/operator/logical_insert.hpp"
```

Comment:
rm `columnstore/columnstore.hpp`, `duckdb/common/enums/operator_result_type.hpp`, `duckdb/planner/operator/logical_insert.hpp`. not needed for header